cmd/hide/main.go

```diff
@@ -87,13 +95,22 @@ func main() {
 	mux.Handle("PUT /projects/{id}/files/{path...}", updateFileHandler)
 	mux.Handle("DELETE /projects/{id}/files/{path...}", deleteFileHandler)
 
+	// TODO: make configurable
 	port := ":8080"
 
-	fmt.Print(Splash)
-	log.Printf("Server started on %s\n", port)
+	log.Info().Msg(fmt.Sprintf("Server started on %s\n", port))
 
 	if err := http.ListenAndServe(port, mux); err != nil {
-		fmt.Println("Error starting server")
-		panic(err)
+		log.Fatal().Err(err).Str("port", port).Msg("Failed to start server")
+	}
+}
+
+func setupLogger(debug bool) {
+	output := zerolog.ConsoleWriter{Out: os.Stdout, TimeFormat: zerolog.TimeFormatUnix, NoColor: false}
```

Comment:
Provides infinite configurability for console formatting. This should be good enough for now.